src/Microsoft.AspNetCore.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -610,6 +674,9 @@ private static class NativeDarwinMonoMethods
             public static extern void uv_unref(UvHandle handle);
 
             [DllImport("__Internal", CallingConvention = CallingConvention.Cdecl)]
+            public static extern int uv_fileno(UvHandle handle, ref IntPtr socket);
```

Comment:
Would end up with mismatched function implements? So if someone did want to use the `fileno` in future calling the function on darwin mono would return incorrect info (if replaced with a dummy function)